bio_hansel/main.py

```diff
@@ -97,6 +100,7 @@ def main():
     init_console_logger(args.verbose)
     output_summary_path = args.output_summary
     output_tile_results = args.output_tile_results
+    output_simple_summary_path = args.output_simple_summary
```

Comment:
Right about here we should add some checking to see if there are already files at each of those paths if the user has supplied those output paths and exit with non-zero status code while alerting user that a file already exists at one of these file paths (we don't want users overwriting their important files). We could add a `--force` opt to allow users to overwrite any files if they know what they are doing.

Reply:
Ah, I didn't think about that. I'll implement a warning.

Reply:
Okay, I've added a check to see if output files already exist, and an optional argument `--force` to overwrite previous output files.